controller/types.go

```diff
@@ -35,6 +35,7 @@ type ChaosResultDetails struct {
 	InjectionTime         float64
 	TotalDuration         float64
 	ChaosEngine           string
+	ChaosEngineLabel	  string
```

Comment:
Can we change the `ChaosEngine           string` to `ChaosEngineName           string`?  It may confuse as we have multiple fields for the engine